### PR TITLE
(refs #29) added fetch citygml url

### PR DESCRIPTION
**Pull request by niryuu**

### CODE REVIEW

**Comment by niryuu** [comment]
gain coverage.

**Comment by smellman** [comment]
- write API document
- add test case with mock
- update README.md

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,13 @@
+certifi==2023.7.22
```
Comment (smellman) [inline requirements.txt:1]: Is it need?

File: requirements.txt [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
+charset-normalizer==3.2.0
```
Comment (smellman) [inline requirements.txt:2]: Is it need?

File: requirements.txt [hunk 3]
```diff
@@ -1,5 +1,13 @@
...
 click==8.1.3
+frozendict==2.3.8
```
Comment (smellman) [inline requirements.txt:4]: Is it need?

File: requirements.txt [hunk 4]
```diff
@@ -1,5 +1,13 @@
...
+idna==3.4
```
Comment (smellman) [inline requirements.txt:5]: Is it need?

File: requirements.txt [hunk 5]
```diff
@@ -1,5 +1,13 @@
...
+httpretty==1.1.4
 numpy==1.24.3
+python-dateutil==2.7.5
```
Comment (smellman) [inline requirements.txt:8]: Is it need?

File: requirements.txt [hunk 6]
```diff
@@ -1,5 +1,13 @@
...
+reearthcmsapi==0.0.2
 requests==2.31.0
 shapely==2.0.1
-tqdm==4.65.0
\ No newline at end of file
+six==1.16.0
```
Comment (smellman) [inline requirements.txt:12]: Is it need?

File: dev-requirements.txt [hunk 7]
```diff
@@ -9,6 +9,7 @@ click==8.1.3
 coverage==7.2.7
 docutils==0.20.1
 exceptiongroup==1.1.1
+httpretty==1.1.4
```
Comment (smellman) [inline dev-requirements.txt:12]: missing reearthcmsapi==0.0.2 for dev-requirements.txt

**Review by smellman — COMMENTED** [review]
dev-requirements.txt generate by `pip freeze > dev-requirements.txt`
requirements.txt is small set for application running.

**Comment by niryuu** [comment]
> dev-requirements.txt generate by `pip freeze > dev-requirements.txt` requirements.txt is small set for application running.

I understood the concept.  I will fix it.

**Review by smellman — APPROVED** [review]
LGTM!

**Pull request merged**